### PR TITLE
👷 Use node slack api on CI

### PR DESCRIPTION
**Pull request by amortemousque**
## Motivation

Use a node slack API instead of slack-notifier image.

## Changes

- Add slack API helper functions
- Use slack API for staging reset job
- Use slack API for bump chrome driver version job
- Use slack API for all CI notification jobs

## Testing

- [x] Local
- [ ] Staging
- [ ] Unit
- [ ] End to end

---

I have gone over the [contributing](https://github.com/DataDog/browser-sdk/blob/main/CONTRIBUTING.md) documentation.

### CODE REVIEW

File: scripts/staging-ci/staging-reset.js [hunk 1]
```diff
@@ -52,6 +59,13 @@ async function main() {
   }
 
   printLog('Reset done.')
+
+  const commitMessage = await executeCommand('git show-branch --no-name HEAD')
```
Comment (BenoitZugmeyer) [inline scripts/staging-ci/staging-reset.js:63]: **🔨 warning:** ‏the HEAD commit message is not the one that triggered the pipeline. It'll be probably the last commit done in this script, `Remove ${CI_FILE} on old branch so pushes are noop`

But I find it a bit weird to have the commit message in scheduled tasks, no? WDYT? 

(Same for the failure message and the chrome driver bump job)

File: .gitlab-ci.yml [hunk 2]
```diff
@@ -256,19 +256,25 @@ deploy-prod-stable:
...
+  extends:
+    - .base-configuration
   before_script:
     - COMMIT_MESSAGE=`git show-branch --no-name HEAD`
     - BUILD_URL="$CI_PROJECT_URL/pipelines/$CI_PIPELINE_ID"
     - COMMIT_URL="$CI_PROJECT_URL/commits/$CI_COMMIT_SHA"
+    # If the pipeline was launched by a user, send pings to them (eg, creating a pipe on master without commiting)
+    - if [[ $GITLAB_USER_LOGIN = "gitsync" ]]; then AUTHOR_EMAIL=$(git show -s --format="%ae" HEAD); else AUTHOR_EMAIL=$GITLAB_USER_EMAIL; fi
+
+notify-feature-branch-failure:
```
Comment (BenoitZugmeyer) [inline .gitlab-ci.yml:270]: **❓ question:** ‏This is great, loving it :) could we do the same on success too?

File: scripts/bump-chrome-driver-version.js [hunk 3]
```diff
@@ -85,7 +91,15 @@ function getMajor(version) {
...
-  await executeCommand(`gh pr create --fill --base ${MAIN_BRANCH}`)
+  const pullRequestUrl = await executeCommand(`gh pr create --fill --base ${MAIN_BRANCH}`)
+  return pullRequestUrl.trim()
 }
 
-main().catch(logAndExit)
+main().catch(async (error) => {
+  const commitMessage = await executeCommand('git show-branch --no-name HEAD')
+  await sendSlackMessage(
+    '##browser-sdk-deploy',
```
Comment (BenoitZugmeyer) [inline scripts/bump-chrome-driver-version.js:101]:
```suggestion
    '#browser-sdk-deploy',
```

File: .gitlab-ci.yml (hunk 2 above)
Comment (amortemousque) [reply .gitlab-ci.yml:278]: Sure, that could be nice ;)

File: scripts/staging-ci/staging-reset.js (hunk 1 above)
Comment (amortemousque) [reply scripts/staging-ci/staging-reset.js]: Yes right. I replaced the commit message by the pipeline id for scheduled tasks

**Review by BenoitZugmeyer — APPROVED** [review]

**Pull request merged**